### ISSUE TITLE
Upcoming Release Versioning

### ISSUE THREAD

**Issue by fredpi** [SFSafeSymbols/SFSafeSymbols#74]
We will probably have a release soon with the big refactoring from #72.
Also, in about a week's time, we will learn about SF Symbols v3 from Apple. I hope that we can implement the changes quite quickly.

I'm wondering which versioning we should use for the upcoming releases. I suggest the following:

- Create a 3.0 release with the big refactoring because there are probably breaking changes which means a major release is needed.
- Create a 3.0.1 release with the new symbols.

Other options:
- Releasing both changes in the same release.
- Using 4.0 as the version for the SF Symbols v3 release.

What do you think? @knothed @ddddxxx @StevenMagdy

**Comment by knothed** [comment]
I'm not sure about the 3.0.0 -> 3.0.1 option as it wrongly suggests users that 3.0.0 already incorporates the v3 SFSymbols. The user expectation is probably that version X.y.z incorporates the version X SFSymbols.

On the other hand, it is bad to artificially restrict the SFSafeSymbol major release cycle to SFSymbol releases. This would forbid any major releases which are not directly linked to a SFSymbol version update.

In conclusion, there is no good solution. The *better* solution is in my opinion to release a minor version now, and then only jump to 3.0.0 with the v3 update.

This makes most sense as #72 is not a breaking change for most users. Only users which depend on certain characteristics of `rawValue` - which they shouldn't be doing in the first place - will probably experience a breaking change. These are an extreme minority.

**Comment by fredpi** [comment]
Closing this because this question is no longer relevant. The 3.0 update will include the refactoring and the new symbols.
